terminal/templates/index.html

```diff
@@ -1,15 +1,15 @@
 <html lang="en">
   <head>
     <link
-      href="{{ url_for('static', filename='xterm/xterm.min.css') }}"
+      href="static/xterm/xterm.min.css"
       rel="stylesheet"
     />
   </head>
   <body style="background-color: black;">
     <div style="width: 100%; height: 100%;" id="terminal"></div>
-    <script src="{{ url_for('static', filename='xterm/xterm.min.js') }}"></script>
-    <script src="{{ url_for('static', filename='xterm/xterm-addon-fit.min.js') }}"></script>
-    <script src="{{ url_for('static', filename='socketio/socket.io.min.js') }}"></script>
-    <script src="{{ url_for('static', filename='terminal.js') }}"></script>
+    <script src="static/xterm/xterm.min.js"></script>
+    <script src="static/xterm/xterm-addon-fit.min.js"></script>
+    <script src="static/socketio/socket.io.min.js"></script>
+    <script src="static/terminal.js"></script>
```

Comment:
what's wrong with `url_for` ?

Reply:
url_for generates an absolute path starting with /static/... because of url rewrite we need a relative path with /terminalXXXX as the prefix. So we get /terminalXXXX/static/...